server/controllers/finance/invoice/lookupConsumableInvoice.js

```diff
@@ -0,0 +1,64 @@
+const db = require('../../../lib/db');
+const identifiers = require('../../../config/identifiers');
+
+/**
+ * @function lookupConsumableInvoicePatient
+ *
+ * @description
+ * this function return an invoice details and all consumable inventories
+ * related to this invoice
+ */
+function lookupConsumableInvoicePatient(req, res, next) {
+  const params = req.query;
+  const record = {};
+
+  let invoiceDetailQuery =
+    `SELECT
+      BUID(invoice.uuid) as uuid, CONCAT_WS('.', '${identifiers.INVOICE.key}',
```

Comment:
In the case of multiple projects, this might produce the incorrect result.  I can't remember why though - just that we found it at HEV.

To retrieve the identifier, it's better to just JOIN on document_map.